pkg/cmd/cli/cmd/set/env.go

```diff
@@ -93,7 +93,7 @@ func NewCmdEnv(fullName string, f *clientcmd.Factory, in io.Reader, out, errout
 	cmd.Flags().StringP("containers", "c", "*", "The names of containers in the selected pod templates to change - may use wildcards")
 	cmd.Flags().StringP("from", "", "", "The name of a resource from which to inject enviroment variables")
 	cmd.Flags().StringP("prefix", "", "", "Prefix to append to variable names")
-	cmd.Flags().StringSliceVarP(&env, "env", "e", env, "Specify key value pairs of environment variables to set into each container.")
+	cmd.Flags().StringArrayVarP(&env, "env", "e", env, "Specify a key value pair of environment variable to set into each container.")
```

Comment:
Specify a key value pair of environment variable -> Specify a key value pair for an environment variable